contents/blog/the-posthog-array.md

```diff
@@ -0,0 +1,58 @@
+---
+date: 2023-02-24
+title: "What’s new in PostHog?"
```

Comment:
```suggestion
title: "PostHog Changelog: February 24, 2023"
```

Reply:
I've taken the date out, as I think we agreed best to have a single article that we update weekly? So we don't want the weekly date in the title...

Reply:
It's preferable to have the date in the title because, when you update each time, it's a strong signal to Google that the page has been updated and it should re-crawl it. Definitely better that way around.